sale_commission/report/sale_commission_analysis_report.py

```diff
@@ -43,22 +43,23 @@ def _get_selection_invoice_state(self):
 
     def _select(self):
         select_str = """
-            SELECT min(aila.id) as id, ai.partner_id partner_id,
-            ai.state invoice_state,
-            ai.date_invoice,
-            ail.company_id company_id,
-            rp.id agent_id,
-            pt.categ_id categ_id,
-            ail.product_id product_id,
-            pt.uom_id,
-            ail.quantity,
-            ail.price_unit,
-            ail.price_subtotal,
-            sc.fix_qty percentage,
-            SUM(aila.amount),
-            ail.id invoice_line_id,
-            aila.settled,
-            aila.commission commission_id
+            SELECT MIN(aila.id) AS id,
+            ai.partner_id AS partner_id,
+            ai.state AS invoice_state,
+            ai.date_invoice AS date_invoice,
+            ail.company_id AS company_id,
+            rp.id AS agent_id,
+            pt.categ_id AS categ_id,
+            ail.product_id AS product_id,
+            pt.uom_id AS uom_id,
+            SUM(ail.quantity) AS quantity,
+            AVG(ail.price_unit) AS price_unit,
+            SUM(ail.price_subtotal) AS price_subtotal,
+            AVG(sc.fix_qty) AS percentage,
+            SUM(aila.amount) AS amount,
+            ail.id AS invoice_line_id,
+            aila.settled AS settled,
+            aila.commission AS commission_id
```

Comment:
This means v11 is bugged too, right?

Reply:
No, v11 is working fine as there's no collision in the column names.

Reply:
@pedrobaeza I was referring to `ail.price_unit` --> `AVG(ail.price_unit) AS price_unit`. Why is this change needed for v12?

Reply:
Well, next time please put the comment in the proper line for correct referral. This is an improvement I have seen for putting the average price, because a sum on this field doesn't make sense IMO.

Reply:
There is not SUM in v11, just `ail.price_unit`. In v11 no aggregate function is used for `quantity`, `price_unit`, `price_subtotal` and `percentage`. This is why I don't understand how  this can work in v11.

Anyway, this PR is OK for me

Reply:
Ah, I remember then: in v11, the SQL view is grouping also by these fields, which is incorrect. It should be fixed in v11 then.